---
int32 total_individual_grasps
int32 total_merged_models